authorization/src/main/kotlin/org/modelix/authorization/permissions/ModelServerSchema.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.authorization.permissions
+
+val MODEL_SERVER_SCHEMA = buildPermissionSchema {
+    resource("model-server") {
```

Comment:
Consider extracting all strings into const variables and use them everywhere instead of the strings.
This makes finding usages much easier.

Reply:
We probably need some generator that generates an API from this schema.

Reply:
Code generation would be nice indeed. But it is not required to get this feature started.

Reply:
Replaced the string constants.